File: Bank/src/main/RegistroMovimientos.java
package main;

import java.text.SimpleDateFormat;
import java.util.Date;

public class RegistroMovimientos {
    private static final String PATTERN = "yyyy-MM-dd HH:mm:ss";
    private static final SimpleDateFormat sdf = new SimpleDateFormat(PATTERN);
    public static String regitry(float saldo){
        String fecha = sdf.format(new Date());
        return fecha +" Saldo Inicial por $"+(int)saldo;
    }
    public static String regitry(float saldo,float cantidad){
        String fecha = sdf.format(new Date());
        return fecha +" Retiro de efectivo por $"+(int)(saldo - cantidad)+", saldo anterior $"+(int)saldo;
    }
    public static String regitry(String fallo){
        String fecha = sdf.format(new Date());
        return fecha +" "+ fallo;
    }
}

File: BankPy/RegistroMovimientos.py
from datetime import datetime


def registry(op, saldo, cantidad):
    tiempo = datetime.now()
    fecha = tiempo.strftime("%Y-%m-%d %H:%M:%S")
    if op == 1:
        return f"{fecha} Saldo incial por ${saldo}"
    elif op == 2:
        return f"{fecha} Retiro de efectivo por ${int(saldo - cantidad)}, saldo anterior ${int(saldo)}"
    elif op == 3:
        return f"{fecha} Operacion Negada"
